src/propagation.cpp

```diff
@@ -132,7 +132,7 @@ static opentracing::expected<void> InjectSpanContextSingleKey(
     const std::unordered_map<std::string, std::string>& baggage) {
   std::ostringstream ostream;
   auto result = InjectSpanContext(PropagationOptions{}, ostream, trace_id,
-                                  sampled, span_id, baggage);
+                                  span_id, sampled, baggage);
```

Comment:
(If only C++ didn't have to respect so many legacies....)

Ryan, it's your call, but I would not object to some kind of type-safe wrapper around the `sampled` bit to avoid this kind of static error.